src/camera.rs

```diff
@@ -65,7 +65,11 @@ pub fn camera_follow_player(
     mut camera_query: Query<(&mut Transform, &Panning), (With<Camera>, Without<Player>)>,
     mut move_event_writer: EventWriter<ParallaxMoveEvent>,
 ) {
-    let player = player_query.single().translation;
+    let player = if let Ok(player) = player_query.get_single() {
```

Comment:
I agree with these checks using .get_single() instead of .single() if we are loading scenes or if players get despawned it would panic otherwise, but can this be shortened to
```rust
if let Ok(player) = player_query.get_single(){
//rest of system goes here
//I also advocate for using player.translation in place of everywhere player is used in this system
}

Reply:
I could see a case for not doing it this way because of rightward drift but it seems more straightforward.

Also eventually we will want to come back and replace all the .get_single() for players when we start having more than 1 player although I imagine at least our camera handling will have to change at that point.